apps/mosaico/src/Article/Article.purs

```diff
@@ -137,9 +113,10 @@ render props =
                       ]
                   }
             ]
-          , DOM.div
-              { className: "mosaico-article__main-image"
-              , children: [ foldMap renderImage mainImage ]
+          , articleMainImage
+              { clickable: true
+              , params: Just "&width=960&height=540&q=90"
+              , image: fold mainImage
```

Comment:
Didn't know one could `fold` a record without an explicit `Monoid` instance 😃

Anyway, this must create a record of
```
  { url: ""
  , caption: Nothing
  , thumb: ""
  , alignment: Nothing
  , byline: Nothing
  }
```
and I don't know if we want that. This would, for instance, create urls with only the query params:
```
<img src="&width=960&height=540&q=90"></img>
```

I suggest we don't even call `articleMainImage` if `mainImage` is `Nothing`. We can use `foldMap`, instead of `fold`, for this.

Reply:
This is indeed the record that gets created: [`fold` works on a `Monoid` instance](https://pursuit.purescript.org/packages/purescript-foldable-traversable/5.0.1/docs/Data.Foldable#v:fold), which `Record` has, and which will happily call the `Monoid` instance on all its fields, see [here](https://github.com/purescript/purescript-prelude/blob/37717776a3693fd5a0a614239f7de1285cba8cdf/src/Data/Monoid.purs#L98) for the implementation